src/index.tsx

```diff
@@ -9,11 +9,28 @@ import { components } from './apiTypes';
 import subscribeCustomer from './actions/subscribeCustomer';
 import getActiveSubscription from './actions/getActiveSubscription';
 import hasPaymentInfo from './actions/hasPaymentInfo';
+
+// Hooks
+import useActiveSubscription from 'hooks/useActiveSubscription';
+import useHasPaymentInfo from 'hooks/useHasPaymentInfo';
+import usePaymentMethodStatus from 'hooks/usePaymentMethodStatus';
+import usePricePlans from 'hooks/usePricePlans';
+import useStripeSetupIntent from 'hooks/useStripeSetupIntent';
+import useUpdateSubscription from 'hooks/useUpdateSubscription';
+
 export const Actions = {
   subscribeCustomer,
   getActiveSubscription,
   hasPaymentInfo,
 };
+export const Hooks = {
+  useActiveSubscription,
+  useHasPaymentInfo,
+  usePaymentMethodStatus,
+  usePricePlans,
+  useStripeSetupIntent,
+  useUpdateSubscription,
+};
```

Comment:
This would mean people have to do something like this:
```ts
import { Hooks } from "octane-components";
const { useActiveSubscription } = Hooks;
```
This has two issues:
1. You have to have that second destructuring line. Since a hook has to be a function that starts with `use`, I don't know that React would like us doing `Hooks.useActiveSubscription()`.
2. Putting the hooks in one object means that we're forcing people to import all of the hooks, even if they use a few. That means that tree shaking can't prune out the unused hooks. That's maybe not a problem now, but that could add up to a lot of JS over time. 

Instead, I'd do something like this above:
```ts
export useActiveSubscription from 'hooks/useActiveSubscription`;
```
That will import and then immediately re-export that module. 

The `Actions` object I have there is obviously not great to have, but we may as well leave that for now.

Reply:
Fixed!